tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -655,6 +655,40 @@ public function testCreateTypeOptions($type, $method, $override)
     }
 
     /**
+     * Test using template vars in Create (formStart template)
+     *
+     * @dataProvider requestTypeProvider
+     * @return void
+     */
+    public function testCreateTemplateVars($type, $method, $override)
```

Comment:
Testing whether templateVars are used by Form::create() doesn't require data provider for type, method etc. Just use a single hardcoded value for these and remove the method arguments and `@dataProvider` tag